.gitignore

```diff
@@ -2,3 +2,8 @@ dist/
 node_modules/
 result
 worker/
+
+# misc & test files
+success-with-server/
+old-success/
+old-error/
```

Comment:
Also, this was re-added, even though this is super specific to your setup. Since we are not using old hpstatus anymore, could this be deleted?

Reply:
@yegortimoshenko - I'll update the .gitignore.  Also, I am aware  the `<base36-id>.holohost.net/api/v1`  endpoint isn't yet in place, but was under the impression it would be soon.  Is this true?  ..or would you suggest making a call to `<base36-id>.holohost.net/api/v1/config` instead?